### DISCUSSION TITLE
QoL suggestion

### DISCUSSION THREAD

**Discussion by TechTonic02** [Malcolm-Q/LC-LateGameUpgrades#545] [Ideas]
It would be nice if you can add a way to show which upgrades are individual, I have times where my friends buy an upgrade not knowing it was only applied to them

**Comment by WhiteSpike** [comment]
![image](https://github.com/user-attachments/assets/09bb1890-c534-4b80-8d80-d9263fd86937)
Was thinking of splitting the upgrades between two lists, one for individual and other for shared, maybe this would work on your case?

**Reply by WhiteSpike** [reply]
Others also suggested of just marking the upgrade as such, maybe in the title text or when selecting the upgrades. It's really a matter of preference of what people want to see.

**Reply by TechTonic02** [reply]
Honestly, both are great, but I am liking the one sent with the picture above, seems like a simple and good of way of separating shared and individual upgrades

**Reply by WhiteSpike** [reply]
Should be present in 3.10.3 release.
